aips/create_dips_job.py

```diff
@@ -186,6 +188,14 @@ def filter_aips(aips, location_uuid):
         if aip["current_location"] != location:
             LOGGER.debug("Skipping AIP (different location): %s", aip["uuid"])
             continue
+        if origin_pipeline_uuid:
+            pipeline = "/api/v2/pipeline/{}/".format(origin_pipeline_uuid)
+            if "origin_pipeline" not in aip:
+                LOGGER.debug("Skipping AIP (missing pipeline): %s", aip["uuid"])
+                continue
+            if aip["origin_pipeline"] != pipeline:
+                LOGGER.debug("Skipping AIP (different pipeline): %s", aip["uuid"])
```

Comment:
I might consider:
```python
if aip.get("origin_pipeline") != pipleine:
   continue
```
But you do get granular debug information your way so that might be more important.